http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/GKEInterpreter.scala

```diff
@@ -405,6 +434,65 @@ class GKEInterpreter[F[_]: Parallel: ContextShift: Timer](
       )
     } yield loadBalancerIp
 
+  private[util] def installGalaxy(appName: AppName,
+                                  dbCluster: KubernetesCluster,
+                                  googleCluster: Cluster,
+                                  nodepoolName: NodepoolName,
+                                  namespaceName: NamespaceName,
+                                  userEmail: WorkbenchEmail)(
+    implicit ev: ApplicativeAsk[F, AppContext]
+  ): F[Unit] =
+    for {
+      ctx <- ev.ask
+
+      _ <- logger.info(
+        s"Installing helm chart ${config.galaxyAppConfig.chart.value} in cluster ${dbCluster.id} | trace id: ${ctx.traceId}"
+      )
+
+      releaseName = ReleaseName(s"${appName.value}-${config.galaxyAppConfig.releaseNameSuffix.value}")
+      chartValues = buildGalaxyChartOverrideValuesString(appName, releaseName, dbCluster, nodepoolName, userEmail)
+
+      _ <- logger.info(
+        s"Chart override values are: ${chartValues} | trace id: ${ctx.traceId}"
+      )
+
+      // The helm client requires a Google access token
+      _ <- F.delay(credentials.refreshIfExpired())
+
+      // The helm client requires the ca cert passed as a file - hence writing a temp file before helm invocation.
+      caCertFile <- writeTempFile(s"gke_galaxy_ca_cert_${dbCluster.id}",
+                                  Base64.getDecoder.decode(googleCluster.getMasterAuth.getClusterCaCertificate),
+                                  blocker)
+
+      helmAuthContext = AuthContext(
+        org.broadinstitute.dsp.Namespace(namespaceName.value),
+        org.broadinstitute.dsp.KubeToken(credentials.getAccessToken.getTokenValue),
+        org.broadinstitute.dsp.KubeApiServer("https://" + googleCluster.getEndpoint),
+        org.broadinstitute.dsp.CaCertFile(caCertFile.toAbsolutePath)
+      )
+
+      // Invoke helm
+      _ <- helmClient
+        .installChart(
+          org.broadinstitute.dsp.Release(releaseName.value),
+          org.broadinstitute.dsp.Chart(config.galaxyAppConfig.chart.value),
+          org.broadinstitute.dsp.Values(chartValues)
+        )
+        .run(helmAuthContext)
+
+      // Poll galaxy until it starts up
```

Comment:
Do we need a TODO to check the applicable Galaxy pod(s) as well when the dependencies are in place?

Reply:
Yeah - I added a TODO in `HttpGalaxyDAO` but maybe it belongs here.